core/src/main/java/org/datasyslab/geospark/formatMapper/FormatMapper.java

```diff
@@ -202,10 +202,28 @@ public Geometry readGeoJSON(String geoJson)
         return null;
     }
 
-    public List<String> readPropertyNames(String geoString) {
+    public boolean containsId(String geoJson){
+        if (geoJson.contains("id")){
+            return true;
+        }
+        else{
+            return false;
+        }
+    }
+
+
+    public List<String> readColumnNames(String geoString) {
+        List<String> columns = new ArrayList<>();
         switch (splitter){
             case GEOJSON:
-              return readGeoJsonPropertyNames(geoString);
+                if(containsId(geoString)){
+                    columns.add("id");
```

Comment:
The logic here is very simple. Please don't create `containsId` function to check geoString. Just simply check it here.

My concern is, `id` is a pretty common word which may exist in other places of this GeoJSON string. Simply using string contain to check this property may not be safe and can result in another bug.

Reply:
Thank you for your detailed comments.
I changed the implementation and using GetId method.

Reply:
Thank you for your comments. 
I try to change my implementation using getId method.